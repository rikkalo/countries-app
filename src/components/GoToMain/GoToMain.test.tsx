import { shallow } from "enzyme";
import React from "react";

import { GoToMain } from "./index";

describe("GoToMain", () => {
  it("should render correctly", () => {
    const wrapper = shallow(<GoToMain />);

    expect(wrapper).toMatchSnapshot();
  });
});
