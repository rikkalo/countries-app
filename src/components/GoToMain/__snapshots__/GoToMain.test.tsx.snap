// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoToMain should render correctly 1`] = `
<WithStyles(ForwardRef(Container))
  maxWidth="sm"
>
  <WithStyles(ForwardRef(Typography))
    component="h3"
    variant="h3"
  >
    <Link
      to="/"
    >
      Go to the main page
    </Link>
  </WithStyles(ForwardRef(Typography))>
</WithStyles(ForwardRef(Container))>
`;
